packages/middleware-sdk-ec2/CHANGELOG.md

```diff
@@ -0,0 +1,82 @@
+# Change Log
```

Comment:
Should this history from the old middleware-ec2-copysnapshot package get carried over?

Reply:
Not required as the package name is changing.
Also, the version numbers shared in the history are not associated with new package name.

You can retain it, as it might be there in other middleware packages whose names have changed.
We can delete it after the CHANGELOG bug is fixed https://github.com/aws/aws-sdk-js-v3/issues/691